Angular 10 support

I got this error when i upgraded to angular 10

> [ng] node_modules/@angular/fire/firestore/firestore.module.d.ts:7:74 - error TS2314: Generic type 'ModuleWithProviders<T>' requires 1 type argument(s).
[ng] 7     static enablePersistence(persistenceSettings?: PersistenceSettings): ModuleWithProviders;

here is the link to the  [Angular migration doc](https://next.angular.io/guide/migration-module-with-providers) for this issue

--- a/src/index.ts
+++ b/src/index.ts
@@ -11,7 +11,7 @@ export { StorageConfig, StorageConfigToken, Storage };
 
 @NgModule()
 export class IonicStorageModule {
-  static forRoot(storageConfig: StorageConfig = null): ModuleWithProviders {
+  static forRoot(storageConfig: StorageConfig = null): ModuleWithProviders<IonicStorageModule> {
     return {
       ngModule: IonicStorageModule,
       providers: [

